src/views/workflow-history/workflow-history-event-status-badge/workflow-history-event-status-badge.constants.ts

```diff
@@ -0,0 +1,26 @@
+import type { WorkflowEventStatusBadgeSize } from './workflow-history-event-status-badge.types';
+
+export const WORKFLOW_EVENT_STATUS = {
+  COMPLETED: 'COMPLETED',
+  FAILED: 'FAILED',
+  ONGOING: 'ONGOING',
+  WAITING: 'WAITING',
+} as const;
```

Comment:
```suggestion
} as const satisfies Record<WorkflowEventStatus, string>;
```
It would be better to have an explicit type here (added more details under .types)